BGL/examples/BGL_OpenMesh/TriMesh.cpp

```diff
@@ -31,18 +31,12 @@ int main(int argc, char** argv )
   Mesh mesh;
 
   std::vector<vertex_descriptor> V;
-  V.push_back(add_vertex(mesh));
-  V.push_back(add_vertex(mesh));
-  V.push_back(add_vertex(mesh));
-  add_face(V.begin(), V.end(), mesh);
-
-  //  OpenMesh::IO::read_mesh(mesh, (argc>1)?argv[1]:"in.off");
-  
+  CGAL::read_off(std::ifstream((argc>1)?argv[1]:"in.off"), mesh);
```

Comment:
`std::ifstream((argc>1)?argv[1]:"in.off")` is an rvalue expression. It cannot bind to `std::istream&`.